src/test/java/org/fundacionjala/coding/daniel/Kata2Test.java

```diff
@@ -0,0 +1,24 @@
+package org.fundacionjala.coding.daniel;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * A class for testing Kata2 class.
+ */
+public class Kata2Test {
+
+    private final Kata2 kata2 = new Kata2();
```

Comment:
this is a bad practice to instantiate object in the attributes of class.
Consider using a @Before method.